### DISCUSSION TITLE
Post to Twitter: Status is a duplicate.

### DISCUSSION THREAD

**Discussion by SheikhKrueger** [jef/streetmerchant#2553] [Help]
Hey,
I've set up all the keys required to auto post on Twitter and everything works fine but I am getting an error which prevents the system from tweeting again. For whatever reason it won't let it post on Twitter.
Discord Notifications work perfectly fine and are sent every time unlike the Twitter Post.
Pls help.  
![dwq](https://user-images.githubusercontent.com/84353417/118558958-97217980-b767-11eb-853c-8b309e17cc1e.PNG)

**Comment by Jwoods2019** [comment]
I would like to know the answer to this as well so i did not start another thread hope some can help with this.

**Comment by lozbinks** [comment]
Probably posting the same thing as the last tweet it sent which is usually blocked for a while by twitter to prevent spam.

**Comment by gjj07** [comment]
Has anyone been able to setup twitter notifications successfully?

**Comment by jef** [comment]
it's probably the way the code is. i will play with it in the future.
